docs/docs/cmd/spo/listitem/listitem-roleinheritance-reset.md

```diff
@@ -0,0 +1,39 @@
+# spo listitem roleinheritance reset
+
+Restores the role inheritance of list item, file, or folder
+
+## Usage
+
+```sh
+m365 spo listitem roleinheritance reset [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site from which the item role inheritance should be reset
+
+`--listItemId <listItemId>`
+: ID of the item to retrieve
```

Comment:
Since we're not retrieving an item, it would be clearer to say: `ID of the item for which to reset role inheritance`